cmd/boots/main.go

```diff
@@ -220,15 +221,52 @@ func main() {
 	}
 }
 
-// getWorkflowFinder returns a no-op workflow finder if tinkerbell is not the backend
-func getWorkflowFinder() (client.WorkflowFinder, error) {
-	dataModelVersion := os.Getenv("DATA_MODEL_VERSION")
-	switch dataModelVersion {
+func getFinders(l log.Logger, c *config) (client.WorkflowFinder, client.HardwareFinder, error) {
+	var hf client.HardwareFinder
+	var wf client.WorkflowFinder = &client.NoOpWorkflowFinder{}
+	var err error
+
+	switch os.Getenv("DATA_MODEL_VERSION") {
```

Comment:
Have we a 'todo' somewhere, or on someones list, to figure out wiring of configuration in Boots generally?

@jacobweinstock @micahhausler

Reply:
Created #618